### ISSUE TITLE
Mono.Linker.MarkException: Error processing method: 'System.String SQLite.SQLite3::GetErrmsg(SQLitePCL.sqlite3)' in assembly: 'SQLite-net.dll' ---> Mono.Cecil.ResolutionException: Failed to resolve System.String 

### ISSUE THREAD

**Issue by candidodmv** [praeclarum/sqlite-net#861]
I'm getting the folling error when try compile Xamarin.Android project. I'm with linker configured only for SDK Assemblies.

Gravidade	Código	Descrição	Projeto	Arquivo	Linha	Estado de Supressão
Erro		MSB4018: Falha inesperada da tarefa "LinkAssemblies".
Mono.Linker.MarkException: Error processing method: 'System.String SQLite.SQLite3::GetErrmsg(SQLitePCL.sqlite3)' in assembly: 'SQLite-net.dll' ---> Mono.Cecil.ResolutionException: Failed to resolve System.String SQLitePCL.raw::sqlite3_errmsg(SQLitePCL.sqlite3)
   em Mono.Linker.Steps.MarkStep.HandleUnresolvedMethod(MethodReference reference)
   em Mono.Linker.Steps.MarkStep.MarkMethod(MethodReference reference)
   em Mono.Linker.Steps.MarkStep.MarkInstruction(Instruction instruction)
   em Mono.Linker.Steps.MarkStep.MarkMethodBody(MethodBody body)
   em Mono.Linker.Steps.MarkStep.ProcessMethod(MethodDefinition method)
   em Mono.Linker.Steps.MarkStep.ProcessQueue()
   --- Fim do rastreamento de pilha de exceções internas ---
   em Mono.Linker.Steps.MarkStep.ProcessQueue()
   em Mono.Linker.Steps.MarkStep.ProcessPrimaryQueue()
   em Mono.Linker.Steps.MarkStep.Process()
   em MonoDroid.Tuner.MonoDroidMarkStep.Process(LinkContext context)
   em Mono.Linker.Pipeline.ProcessStep(LinkContext context, IStep step)
   em Mono.Linker.Pipeline.Process(LinkContext context)
   em MonoDroid.Tuner.Linker.Process(LinkerOptions options, ILogger logger, LinkContext& context)
   em Xamarin.Android.Tasks.LinkAssemblies.Execute(DirectoryAssemblyResolver res)
   em Xamarin.Android.Tasks.LinkAssemblies.Execute()
   em Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   em Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()		 	0

**Comment by candidodmv** [comment]
I don't know if it's relevant information, but the project has previously Akavache in use. After install sqlite-net-pcl . I'm starting getting the above error.

**Comment by ericsink** [comment]
It seems likely that any project using both Akavache and sqlite-net might need to remain on Akavache 6.5.x until sqlite-net transitions to SQLitePCLRaw 2.0. See #860

**Comment by candidodmv** [comment]
Yes, Unfortunately I didn't saw the referral link above, I discover this incompatibility through hard way.

**Comment by praeclarum** [comment]
Working on this!

**Comment by PicOLinO** [comment]
@praeclarum, I see that this bug fixed in v1.7+ version (#860), can you assume when next release will be?

**Comment by dlpopescu** [comment]
I have the same issue, but on iOS, no Akavache.
Below, the sqlite libraries currently installed in the ios project.

![Screenshot 2019-10-08 at 15 20 59](https://user-images.githubusercontent.com/21035866/66395127-597f3b00-e9df-11e9-9f35-6026c1a33dfe.png)

**Comment by ericsink** [comment]
@dlpopescu 

SQLitePCLRaw 2.x is compatible with sqlite-net-pcl 1.7.

SQLitePCLRaw 1.x is compatible with sqlite-net-pcl 1.6.

Also, don't mix pieces of SQLitePCLRaw 1.x and 2.x at the same time.

**Comment by dlpopescu** [comment]
@ericsink : thank you, much appreciated.
